pkg/reconciler/kubernetes/tektoninstallerset/const.go

```diff
@@ -21,4 +21,12 @@ const (
 	CreatedByKey       = "operator.tekton.dev/created-by"
 	ReleaseVersionKey  = "operator.tekton.dev/release-version"
 	TargetNamespaceKey = "operator.tekton.dev/target-namespace"
+	InstallerSetType   = "operator.tekton.dev/type"
+
+	ClusterTaskInstallerSet            = "ClusterTask"
+	PipelinesTemplateInstallerSet      = "PipelinesTemplate"
+	TriggersResourcesInstallerSet      = "TriggersResources"
+	ConsoleCLIInstallerSet             = "ConsoleCLI"
+	MiscellaneousResourcesInstallerSet = "MiscellaneousResources"
+	CreatedByValue                     = "TektonAddon"
```

Comment:
these are addon specific const right?
should we keep them in addon pkg? and not in tektoninstallerset?